qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddNewRolePage.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class AddNewRolePage extends Page{
+
+	protected static final String PAGE_URL = "/openmrs/admin/users/role.form";
```

Comment:
is AddNewRole a page or a form?

Reply:
it's a page however much it has a form tag embedded

Reply:
isn't it a form that can be accessed through ManagePriviligePage? What happens when you instead add 

```js
public void addRoleName(String roleName){
		findElement(ROLE_NAME_FIELD).clear();
		findElement(ROLE_NAME_FIELD).sendKeys(roleName);
	}

	public void addDescription(String description){
		findElement(DESCRIPTION_FIELD).clear();
		findElement(DESCRIPTION_FIELD).sendKeys(description);
	}

	public void selectPrivileges(){
		clickOn(SELECT_OR_UNSELECT_ALL_PRIVILEGES);
	}

	public void saveRole(){
		clickOn(SAVE_BUTTON);
	}
```
to ManagePrivilegePage and get of rid of this file?

Reply:
Hope you have not forgotten for Roles am using the legacy ui(am going through the `systemAdministrationPage` and `administrationPage`) unlike the privileges where am going through `configuremetadataPage` but let me try it and come back with a response as per the review above. Thanks

Reply:
Adding the elements above on [ManagePrivilegePage](https://github.com/openmrs/openmrs-contrib-qaframework/blob/71955f33736fc30263c56276fdc0000293457fff/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManagePrivilegesPage.java).

Reply:
i have done that but its not achievable on my end.

Reply:
how are you doing it?

Reply:
well like you said, i got [this](https://github.com/openmrs/openmrs-contrib-qaframework/pull/270#discussion_r770375178) and added it to the `managePrivilegesPage` alongside the [selectors starting from here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/71955f33736fc30263c56276fdc0000293457fff/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManagePrivilegesPage.java#L18) ofcos. Then changed the steps in reference to that so that instead of using `addNewRolePage`, they use `managePrivilegesPage`